bin/test-workflow/start

```diff
@@ -3,16 +3,108 @@
 set -eo pipefail
 cd "$(dirname "$0")" || ( echo "cannot cd into dir" && exit 1 )
 
-./0_prep_conjur_in_kind.sh
+function print_help {
+  cat << EOF
+Runs the end-to-end workflow for testing Conjur Kubernetes authentication.
+By default, the workflow is run against Conjur Open Source, but can be run
+against Conjur Enterprise.
 
-source ./1_prep_env.sh
+Usage: ./start [options]:
 
-./2_admin_load_conjur_policies.sh
-./3_admin_init_conjur_cert_authority.sh
+    -o, --open-source         Run the E2E workflow against Conjur Open Source
+    -e, --enterprise          Run the E2E workflow against Conjur Enterprise
```

Comment:
Do we need both `--open-source` and `--enterprise` options? Or are these added to make the workflow choice  obvious/explicit when the script is invoked?
Probably not a big deal, but if both options are chosen, the one entered last will take precedence. Also, someone might mistakenly assume that both can be run in parallel by choosing both options?

Would be simpler to only support the `--enterprise` option?

Reply:
Good idea! My intention was to make it explicit, but I overlooked one overriding the other.